crates/state/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "svm-state"
+version = "0.1.0"
+edition = "2018"
+publish = false
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1"
+blake3 = "1"
+funty = "=1.1.0"
+futures = "0.3"
+redis = { version = "0.21", features = ["tokio-comp"] }
```

Comment:
Why redis?

Reply:
Ok, it seems not to be used (the same for `funty` and `futures`)

Reply:
`funty` is needed to avoid breakage with `nom` and `bitvec` (see https://github.com/myrrlyn/funty/issues/3). I've moved `futures` to `[dev-dependencies]`.

`redis` indeed is not needed, a remnant of some old prototyping.